project.mk

```
@@ -0,0 +1,10 @@
+# Project specific values
+OPERATOR_NAME?=managed-upgrade-operator
+OPERATOR_NAMESPACE?=managed-upgrade-operator
+
+IMAGE_REGISTRY?=quay.io
+IMAGE_REPOSITORY?=openshift-sre
```

I see other SRE operators (gcp, cloud-ingress, dms, aws) tend to do:
```suggestion
IMAGE_REPOSITORY?=$(USER)
```
here.. which I guess assists when doing personal development. Could we do a similar thing here?

---

Ok, I understand your point. Yeah we could do that as well. The eventual operator deployment image will be from a standard location as such so this could help with dev stuff..

---

Have added the change